lib/OpenQA/Task/Needle/Save.pm

```diff
@@ -41,6 +39,11 @@ sub _json_validation {
         die 'area without type' unless exists $area->{type};
         die 'area without height' unless exists $area->{height};
         die 'area without width' unless exists $area->{width};
+        if ($area->{type} eq 'ocr') {
+          die 'OCR area without refstr' unless exists $area->{refstr};
```

Comment:
Since this is later used in a string comparison it better not only exist but also be defined:
```suggestion
          die 'OCR area without refstr' unless defined $area->{refstr};
```